packages/tldraw/src/lib/defaultExternalContentHandlers.ts

```diff
@@ -440,6 +442,45 @@ export function registerDefaultExternalContentHandlers(
 	})
 }
 
+/** @public */
+export async function createMediaAssetInfoSkeleton(
+	file: File,
+	assetId: TLAssetId,
+	isImageType: boolean,
+	isVideoType: boolean
+) {
```

Comment:
> InfoSkeleton -> just Info?

I added skeleton to call out that the object sent back is partial (missing src) — I'll rename to add `Partial`

> Can we infer isImageType and isVideoType from the File if we move this onto the editor and have it look at the accepted asset type options? If not, can we pass these in as an options bag instead of floating booleans?

Unfortunately not :-/ The problem is that there is currently two callers to the function. One relies on `acceptedImageMimeTypes`/`acceptedVideoMimeTypes` and the other relies on sniffing the File.

I updated the rest - thx!